pkg/csi/service/common/constants.go

```diff
@@ -429,10 +429,13 @@ const (
 	// WorkloadDomainIsolationFSS is FSS for Workload Domain isolation feature
 	// Used in PVCSI
 	WorkloadDomainIsolationFSS = "workload-domain-isolation"
+	// VPCCapabilitySupervisor is a supervisor capability indicating if VPC FSS is enabled
+	VPCCapabilitySupervisor = "VPC_Supported"
```

Comment:
Do you have plan for making change in PVCSI for this capability?
If yes then you might need CSI FSS as well.

Reply:
No changes are required in PVCSI.